tests/README.md

```diff
@@ -32,6 +32,8 @@ export PATH=$BINDIR:$PATH
 export LD_LIBRARY_PATH=$LIBDIR
 
 python3 ted.py
+# or
+python3 ted.py --device /dev/dri/renderD128  # to specify device directly
```

Comment:
add comment to default path:

```
python3 ted.py # will execute on default device (<...>)
# or
python3 ted.py --device /dev/dri/renderD128  # to specify device directly
```

Also, what "default device" means? Do we explicitly specify /dev/dri/renderD128 or rely on apps device selection - this also needs to be reflected in the above comment...

Reply:
If we don't provide `--device` flag script automatically use `/dev/dri/renderD128` as default value. If we provide device explicitly we use provided one.
```
python3 ted.py # uses /dev/dri/renderD128 (implicitly, we set this device as default without any specific checks)
python3 ted.py --device /dev/dri/renderD129 # uses /dev/dri/renderD129
```